x-pack/plugins/index_lifecycle_management/__jest__/client_integration/helpers/setup_environment.ts

```diff
@@ -29,6 +29,7 @@ export const setupEnvironment = () => {
   );
 
   mockHttpClient.interceptors.response.use(({ data }) => data);
+  // @ts-ignore
```

Comment:
Why is this necessary? It seems odd that the other changes in this PR would manifest a TS error here.